diff -p -up jabberd-2.2.8/etc/c2s.xml.dist.in.pid jabberd-2.2.8/etc/c2s.xml.dist.in
--- jabberd-2.2.8/etc/c2s.xml.dist.in.pid	2009-05-26 23:00:35.000000000 +0200
+++ jabberd-2.2.8/etc/c2s.xml.dist.in	2009-05-26 23:01:17.000000000 +0200
@@ -5,7 +5,7 @@
 
   <!-- The process ID file. Comment this out if you don't need to know
        the process ID from outside the process (eg for control scripts) -->
-  <pidfile>@localstatedir@/@package@/pid/c2s.pid</pidfile>
+  <pidfile>/var/run/jabberd/c2s.pid</pidfile>
 
   <!-- Router connection configuration -->
   <router>
diff -p -up jabberd-2.2.8/etc/router.xml.dist.in.pid jabberd-2.2.8/etc/router.xml.dist.in
--- jabberd-2.2.8/etc/router.xml.dist.in.pid	2009-05-26 22:58:56.000000000 +0200
+++ jabberd-2.2.8/etc/router.xml.dist.in	2009-05-26 23:00:07.000000000 +0200
@@ -5,7 +5,7 @@
 
   <!-- The process ID file. Comment this out if you don't need to know
        the process ID from outside the process (eg for control scripts) -->
-  <pidfile>@localstatedir@/@package@/pid/router.pid</pidfile>
+  <pidfile>/var/run/jabberd/router.pid</pidfile>
 
   <!-- Log configuration - type is "syslog", "file" or "stdout" -->
   <log type='syslog'>
diff -p -up jabberd-2.2.8/etc/s2s.xml.dist.in.pid jabberd-2.2.8/etc/s2s.xml.dist.in
--- jabberd-2.2.8/etc/s2s.xml.dist.in.pid	2009-05-26 23:01:36.000000000 +0200
+++ jabberd-2.2.8/etc/s2s.xml.dist.in	2009-05-26 23:02:06.000000000 +0200
@@ -5,7 +5,7 @@
 
   <!-- The process ID file. Comment this out if you don't need to know
        the process ID from outside the process (eg for control scripts) -->
-  <pidfile>@localstatedir@/@package@/pid/s2s.pid</pidfile>
+  <pidfile>/var/run/jabberd/s2s.pid</pidfile>
 
   <!-- Router connection configuration -->
   <router>
diff -p -up jabberd-2.2.8/etc/sm.xml.dist.in.pid jabberd-2.2.8/etc/sm.xml.dist.in
--- jabberd-2.2.8/etc/sm.xml.dist.in.pid	2009-05-26 23:02:22.000000000 +0200
+++ jabberd-2.2.8/etc/sm.xml.dist.in	2009-05-26 23:02:55.000000000 +0200
@@ -7,7 +7,7 @@
 
   <!-- The process ID file. Comment this out if you don't need to know
        the process ID from outside the process (eg for control scripts) -->
-  <pidfile>@localstatedir@/@package@/pid/sm.pid</pidfile>
+  <pidfile>/var/run/jabberd/sm.pid</pidfile>
 
   <!-- Router connection configuration -->
   <router>
